packages/accounts-2fa/utils.js

```diff
@@ -0,0 +1,12 @@
+import twofactor from "node-2fa";
+import QRCode from "qrcode-svg";
+
+export const generateSecret = ({ username, appName } = {}) => {
+  const { secret, uri } = twofactor.generateSecret({ name: appName, account: username });
```

Comment:
If we assume that this package is only going to be used together with `accounts-password`, then this is fine, but it runs into trouble when OAuth packages are concerned as sometimes you might now have `username` available, would be better to be able to use user ID as well.

Reply:
Yes, there is a lot of changes we need to here. It's far from done. This was just a test. I'll work on this again next week 😉